requirements/edx/base.txt

```diff
@@ -12,6 +12,10 @@
     # via -r requirements/edx/github.in
 -e git+https://github.com/edx/django-wiki.git@1.0.2#egg=django-wiki
     # via -r requirements/edx/github.in
+-e git+https://github.com/edx/edx-ace.git@AA-910#egg=edx-ace
```

Comment:
all the requirements changes are temporary .. i'm just getting the library from edx-ace to see if tests pass with my branch - when that is merged this would be changed to the new edx-ace tag